Simplenote/NoteTableCellView.swift

```diff
@@ -249,14 +266,30 @@ extension NSAttributedString {
     /// Returns a NSAttributedString representation of a given String, with the specified parameters.
     /// List Markers will be replaced by Text Attachments
     ///
-    static func previewString(text: String, font: NSFont, color: NSColor) -> NSAttributedString {
+    static func previewString(text: String,
+                              font: NSFont,
+                              color: NSColor,
+                              highlighting highlight: (keywords: [String]?, color: NSColor, font: NSFont)? = nil) -> NSAttributedString {
```

Comment:
I'm wondering if it's worth encapsulating these 3 properties into a struct (`struct KeywordHighlight`?), or dropping the tuple?

(Just to improve a bit readability!)

Reply:
At first I did it without a tuple, but we're not using highlighting in all cases, so `color` and `font` for highlighting would need to be optional. To avoid this I wrapped it in a tuple and made the tuple optional.
I think it looks pretty much like struct. Why do you think it's not very readable, because of the line length?

Reply:
Yeah it's just the line length. BUT it's a cosmetic detail, so TBH no need to change anything, I'd say.

A typealias maybe is what we're thinking of?

Reply:
Great thinking! Added a typealias :-)